ibraheemosule/skoolizy
# za - modify banner

# Why this PR?
modified banner

# What does this PR do?
modifies the banner for sign in and sign out

# How to test your work?
click on login and a banner should display for a successful sign in and likewise for sign out

# Pages to see your changes?
/auth/login
